### PR TITLE
Add unifyEitherT function.

### PR DESCRIPTION
**Pull request by tmcgilchrist**
@domdere

### CODE REVIEW

File: src/Control/Monad/Trans/Either.hs [hunk 1]
```diff
@@ -56,7 +57,7 @@ import           Control.Monad.Trans.Class (lift)
 import           Control.Monad.Trans.Except (ExceptT(..))
 
 import           Data.Maybe (Maybe, maybe)
-import           Data.Either (Either(..), either)
+import           Data.Either (Either(..), either, first)
```
Comment (domdere) [inline src/Control/Monad/Trans/Either.hs]: `first` comes from Bifunctor

**Review by domdere — CHANGES_REQUESTED** [review]
Just have to fix up the import

**Comment by nhibberd** [comment]
I haven’t hit this issue, I guess I just right hoist firstT instead using transformers-bifunctor

**Comment by domdere** [comment]
hmm yeh maybe I just need to get accustomed to using these: https://hackage.haskell.org/package/transformers-bifunctors-0.1/docs/Control-Monad-Trans-Bifunctor.html#v:firstX

**Comment by domdere** [comment]
@tmcgilchrist Happy to close this btw

**Comment by tmcgilchrist** [comment]
Will add some docs to point off to `transformers-bifunctors` as a supplemental library.

**Pull request closed without merging**